tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```
@@ -170,6 +174,7 @@ spec:
         - "--strict=false"
         - "--show-successes"
         - "--effective-time=$(params.EFFECTIVE_TIME)"
+        - "--extra-rule-data='$(params.EXTRA_RULE_DATA)'"
```

The `args` will be taken as strings and provided as is in the `argv` array to the executable in `command`, this is not subject to shell words, so no need for quotes here. The quotes that remain here are for YAML to force `args` to be an array of strings, but most likely we do not need them as well as they could default to being strings. Good to be safe though...

```suggestion
        - "--extra-rule-data=$(params.EXTRA_RULE_DATA)"
```

---

OK, I reverted the quotes, and it looks like acceptance tests are green again :+1: fyi @simonbaird

---

Ooh now I get it, thanks..!